### ISSUE TITLE
Ansible production playbook fails to output ths/aths values, locking the admin out of servers

### ISSUE THREAD

**Issue by ehartsuyker** [freedomofpress/securedrop#1248]
The playbook `securedrop-prod.yml` ran from head to toe against my two servers, both completely fresh installs of Ubuntu 14.04.3, but at the end, no values for the ths/aths were written to the directory [as described here](https://docs.securedrop.org/en/latest/install.html#run-the-ansible-playbook). I am now locked out of the servers.

My env.
```
Host: Tails 2.0.1
Target: Ubuntu 14.04.3

amnesia@amnesia:~/Persistent/securedrop/install_files/ansible-base$ python --version
Python 2.7.9
amnesia@amnesia:~/Persistent/securedrop/install_files/ansible-base$ ansible --version
ansible 1.7.2
amnesia@amnesia:~/Persistent/securedrop/install_files/ansible-base$ uname -a
Linux amnesia 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt20-1+deb8u3 (2016-01-17) x86_64 GNU/Linux
```

It would be nice to have these values written out with the `debug` module, so that at the very least other admins can look at the `ansible` outout and access their instances, even if the files aren't written to disk on the admin work station.

**Comment by conorsch** [comment]
@ehartsuyker There was no error output during the Ansible run? If the aths-fetching tasks didn't fail, they should absolutely be on your Tails host. Hop up a few directories and run a `find -type f -iname '*-ths'` or similar just to make sure. The files _should_ land adjacent to the `securedrop-prod.yml` playbook, but it's worth checking elsewhere, since Ansible gets creative about relative filepaths.

Were _any_ of the `*-ths*` files fetched?

**Comment by ehartsuyker** [comment]
Yep, relative pathing issue. Close it (sorry).

**Comment by conorsch** [comment]
@ehartsuyker Excellent, thanks for the quick follow-up. So I'm clear on the issue: were you running the `ansible-playbook` command from the repo root (i.e. `~/Persistent/securedrop`)? We recommend that all ansible commands be run from the ansible project root, which is `~/Persistent/securedrop/install_files/ansible-base`, but we could probably be clearer about that in our documentation. I'd like to scan the docs for clarity here, but want to make sure I understand the snag fully first.

**Comment by ehartsuyker** [comment]
I ran it in a different directory, so I guess the docs could be clearer and
say "always run from `ansible-base`."

**Comment by freddymartinez9** [comment]
as far as I know, the admin will not be locked out the servers, the .onion address should still be written in your torrc files but you might have to alias the .onion in your sshd.  You wouldnt be able to login to the app or mon servers, but it _should_ still be reachable. Please correct me if I'm wrong.

**Comment by garrettr** [comment]
@freddymartinez9 Once the SecureDrop installation process is complete, remote access is only possible through an authenticated Tor Hidden Service. If you don't have the HidServAuth cookies, then yes, you will be locked out (although if you have physical access you can still get in).

**Comment by freddymartinez9** [comment]
sure I'm well aware of that, but my point is that just because you don't see the shortcuts doesn't mean you can't access the server. The auth cookie and the onion might still be on the Persistent volume if you grep for '.onion', you might find it...but you still might have to alias in SSH. does that make sense?

**Comment by ehartsuyker** [comment]
@garrettr After the installation was complete and `ansible` placed the output files one dir up from where expected, I plugged in a keyboard and monitor, rebooted, and tried to log in with the user/pass I setup on the OS install, and it fails. I believe they are *only* accessible through the `aths`.

**Comment by garrettr** [comment]
@freddymartinez9 You are correct about that, good point. I think the general takeaway here is that the installation process via Ansible is more difficult, confusing, and easier to mess up than it should be.

**Comment by garrettr** [comment]
@ehartsuyker Sorry, I should have clarified. You can access them in root/rescue mode.

**Comment by garrettr** [comment]
Closing as invalid, this is not a bug, although the installation process could be made easier and more bullet-proof.
